nginx/nginx_test.go

```diff
@@ -165,7 +157,10 @@ func TestFailsIfNginxDiesEarly(t *testing.T) {
 	lb := newUpdaterWithBinary(tmpDir, "./fake_failing_nginx.sh")
 
 	assert.Error(lb.Start())
```

Comment:
Ah right. We should add a comment here for that, since it's not obvious. Ideally the script would handle `-v` but fail when simulating a start.